### PR TITLE
Add HTTP source examples to sideload

### PR DESCRIPTION
**Pull request by kelseiv**
Closes https://github.com/influxdata/docs-v2/issues/2035

### CODE REVIEW

File: content/kapacitor/v1.5/nodes/sideload_node.md [hunk 1]
```diff
@@ -151,16 +153,67 @@ sideload.quiet()
...
+```js
+|sideload()
+  .source('host1')
+  .order('host/{{.host}}.yml', 'hostgroup/{{.hostgroup}}.yml')
+  .field('cpu_threshold', 0.0)
+  .tag('foo', 'unknown')
+```
 
+{{% note %}}
+A source defined as a **URL** or **endpoint** is loaded once as an HTTP GET when a task is enabled, and then on subsequent calls to the `/sideload/reload` endpoint.
```
Comment (sanderson) [inline content/kapacitor/v1.5/nodes/sideload_node.md:193]:
```suggestion
Kapacitor loads **URL** or **endpoint** sources once using an HTTP `GET` requests when a task is enabled, and then on subsequent calls to the `/sideload/reload` endpoint.
```

File: content/kapacitor/v1.5/nodes/sideload_node.md [hunk 2]
```diff
@@ -151,16 +153,67 @@ sideload.quiet()
...
+{{% /note %}}
 
+An HTTP source endpoint should return a JSON where each property is a key name specified in the order statement and its value is an object with a set of key/value pairs.
```
Comment (sanderson) [inline content/kapacitor/v1.5/nodes/sideload_node.md:196]:
```suggestion
An HTTP source endpoint should return a JSON object where each property is a key name specified in the order statement and its value is an object with a set of key-value pairs.
```

We probably need to decide how we want to format "key-value." Is it "key/value" or "key-value". Either way, it should be the same in all the docs.

File: content/kapacitor/v1.5/nodes/sideload_node.md [hunk 3]
```diff
@@ -151,16 +153,67 @@ sideload.quiet()
...
+"host2": {
+"cpu_threshold": 85,
+"disable": "true"
+},
+"host3": {
+"cpu_threshold": 56,
+"disable": "false"
+}
+}
+```
```
Comment (sanderson) [inline content/kapacitor/v1.5/nodes/sideload_node.md:214]:
~~~suggestion
```js
{
    "host1": {
        "cpu_threshold": 99.9,
        "disable": "false"
    },
    "host2": {
        "cpu_threshold": 85,
        "disable": "true"
    },
    "host3": {
        "cpu_threshold": 56,
        "disable": "false"
    }
}
```
~~~

File: content/kapacitor/v1.5/nodes/sideload_node.md [hunk 4]
```diff
@@ -151,16 +153,67 @@ sideload.quiet()
...
+
+<a class="top" href="javascript:document.getElementsByClassName('article-heading')[0].scrollIntoView();" title="top"><span class="icon arrow-up"></span></a>
```
Comment (sanderson) [inline content/kapacitor/v1.5/nodes/sideload_node.md:216]: These don't work anymore and should probably be removed.

**Review by sanderson — CHANGES_REQUESTED** [review]

File: content/kapacitor/v1.5/nodes/sideload_node.md (hunk 2 above)
Comment (kelseiv) [reply content/kapacitor/v1.5/nodes/sideload_node.md:196]: agree, key-value works

File: content/kapacitor/v1.5/nodes/sideload_node.md (hunk 4 above)
Comment (kelseiv) [reply content/kapacitor/v1.5/nodes/sideload_node.md:215]: Appears we have 3453 occurrences in 69 files

**Review by sanderson — APPROVED** [review]

File: content/kapacitor/v1.5/event_handlers/bigpanda.md [hunk 5]
```diff
@@ -37,8 +37,10 @@ Authorization Bearer token for BigPanda REST API.
 
 #### `app-key`
 
+
 BigPanda integration App Key. To get your App Key, log in to your BigPanda account and select **Integrations** in the header toolbar, and then click **New Integration**.
 Select **Alerts REST API**, click **Integrate**, and then **Create an App Key**.
+>>>>>>> master
```
Comment (docmerlin) [inline content/kapacitor/v1.5/event_handlers/bigpanda.md:43]: This looks like someone did a bad merge.

File: content/kapacitor/v1.5/nodes/sideload_node.md [hunk 6]
```diff
@@ -151,16 +153,70 @@ sideload.quiet()
 
 ### Source
 
+<<<<<<< HEAD
+Define the source of the data. The following sources are supported:
+=======
 Source for the data, for example `file://` or `http://`.
+>>>>>>> master
```
Comment (docmerlin) [inline content/kapacitor/v1.5/nodes/sideload_node.md:160]: Bad merge.

**Review by docmerlin — CHANGES_REQUESTED** [review]
Looks like there was a bad merge, that wasn't cleaned up.  It needs to be cleaned up.

**Pull request merged**